### DISCUSSION TITLE
现在还有效吗

### DISCUSSION THREAD

**Discussion by Delerrr** [coupon-clipper/cuttlefish#3] [General]
不了解墨斗鱼，现在还可以用吗

**Comment by liuliangsir** [comment]
> 不了解墨斗鱼，现在还可以用吗

应该可以，只是有些问题 chatgpt 回答不了，请求回答的接口会报异常，最终会生成一个空的 docx 文档

**Reply by codingcc123** [reply]
Error: ChatGPT error 429: {
    "error": {
        "message": "You exceeded your current quota, please check your plan and billing details.",
        "type": "insufficient_quota",
        "param": null,
        "code": null
    }
}    chatGpt接口报这个错，配额不足，没办法接口调了。

**Reply by liuliangsir** [reply]
重新注册一个 chatgpt 账号
